### ISSUE TITLE
Samsung Galaxy Tab 3

### ISSUE THREAD

**Issue by SubJunk** [UniversalMediaServer/UniversalMediaServer#431]
Trying to troubleshoot why my Samsung Galaxy Tab 3 thinks there's no content on my UMS.

I am using fresh build of commit 08f96d1a90bdbdcfeb7d409e4ae3711fc0a343a8 and the built-in "Video" player on the device. It's able to see the UMS server (under "Nearby Devices"), but when I select it, it gives me the message

    Contents location is not available. Check network status and connected device.

I've taken a look at the traffic coming from and going to the device and it goes through the following sequence:

    GET /description/fetch
    GET /UPnP_AV_ContentDirectory_1.0.xml
    GET /UPnP_AV_ConnectionManager_1.0.xml
    SUBSCRIBE /upnp/event/content_directory
    SUBSCRIBE /upnp/event/connection_manager
    POST /upnp/control/content_directory (ContentDirectory:1#GetSearchCapabilities)

UMS seems to reply correctly to each request (and sends NOTIFYs in response to the SUBSCRIBEs). However, the tablet never sends a POST for ContentDirectory:1#Browse. I'm guessing it thinks there's no content because SearchCaps comes back empty, but again, that's my guess.

 from http://www.universalmediaserver.com/forum/viewtopic.php?f=5&t=2614

**Comment by Sami32** [comment]
@SubJunk So, since then what's happened ? Did you found a way to use it ?

**Comment by SubJunk** [comment]
@Sami32 I only had that phone as a temporary replacement for my iPhone so I can't check this anymore. I'll close the issue and if anyone else wants to re-open it if they have a Samsung phone to test it on, go ahead

**Comment by Sami32** [comment]
@SubJunk Thank you very much for the access to the UMS organization, but i was simply think having access to the wiki, so now i really get scared to do mistakes 8-/
I did feel much better when i did know that someone was looking behind my shoulder, i really need don't forget to do only PR ;-)

**Comment by Nadahar** [comment]
@SubJunk I have a Galaxy Tab 3 tablet, I'm guessing it's pretty close to the phone "logically". What I don't understand is why you differentiate between different hardware running Android. Is there really a difference? To me it's like comparing a Dell with Windows 7 to an Acer running Windows 7 - not really relevant when it comes to DLNA support. In addition, I don't know which software is referred to when just the device is mentioned. Is there such a thing as an Android "default" DLNA client? I've used this tablet for testing earlier, but I've then used ```BubbleUPnP``` or some other software doing the DLNA.
